stored-procedures/sp_kmeans/sp_kmeans.sql

```diff
@@ -38,7 +38,7 @@ BEGIN
     -- create kmeans tables and choose random starting centroids
     EXECUTE 'CREATE TABLE IF NOT EXISTS ' || tablename || '_kmeans (
         cluster int, centroid SUPER, startts timestamp, endts timestamp, 
-        interations int) DISTSTYLE ALL';
+        iterations int) DISTSTYLE ALL';
```

Comment:
This change diverges from the posted article https://repost.aws/articles/ARPoweQIN2ROOXZiJAtSQvkQ/vector-search-with-amazon-redshift, so it would be nice if someone could fix the article.

That said, the token as written is meaningless and clarifying even only here will still help readers.